Add Social Security Number to validation

antoiba86 opened the pull request:
### Summary
Add validation for the NNSS.

I upgrade this awesome package by adding another validation for the spanish security social number.

### List

1. Added function in class Identity
2. Added function in class IdentityServiceProvider
3. Added a test funciton
4. Modified README to include the new functionality

antoiba86 committed:
Add function to validate Number of Spanish Social Security

antoiba86 committed:
Add nnss to validator and function to test the NNSS

antoiba86 committed:
Update README to add the validation of NNSS

mpijierro reviewed (commented):
Hello,

Could you extract the assertTrue assertion in function test_invalid_nnss () to a new function test_valid_nnss() ?
I think it would be more orderly.

Thank you and great job!

antoiba86 committed:
Added new function to test valid nnss and invalidad nnss

antoiba86 committed:
Correct test to test nnss is invalid

antoiba86 committed:
Correct Assert to include only false in invalid function

antoiba86 commented:
Hello,
Sorry for the delay, too much work theses days.
I updated the code and added a function test_valid_nnss and test_invalid_nnss.

Thanks to you because you did a great job.
